src/ServiceMonitor/Main.cpp

```diff
@@ -51,10 +51,6 @@ int __cdecl _tmain(int argc, _TCHAR* argv[])
             goto Finished;
         }
 
-        //

```

Comment:
This comment is about environment variables being updated and that behavior being hardcoded.

Reply:
got it, will revert that...